.rspec_parallel

```diff
@@ -0,0 +1,6 @@
+--require spec_helper
```

Comment:
L&D question - I don't quite understand how this get's invoked or why it's needed. It is mainly for formatting, helping with logs and tagging? Can't make sense of [the documentation](https://github.com/grosser/parallel_tests?tab=readme-ov-file#loggers).  

My only other thought is that this set-up help with the coverage report?

Reply:
This isn't actually required as all the tests require 'rails_helper' which includes this. I've taken it out. Its default for rspec, which is not rails-specific